packages/pigment-css-nextjs-plugin/src/index.ts

```diff
@@ -16,8 +16,16 @@ const extractionFile = path.join(
   'zero-virtual.css',
 );
 
-export function withPigment(nextConfig: NextConfig, pigmentConfig: PigmentOptions) {
-  const { babelOptions, asyncResolve, ...rest } = pigmentConfig;
+export function withPigment(nextConfig: NextConfig, pigmentConfig?: PigmentOptions) {
+  const { babelOptions = {}, asyncResolve, ...rest } = pigmentConfig ?? {};
+  if (process.env.TURBOPACK === '1') {
```

Comment:
Is there a link to where `TURBOPACK` is coming from?

Reply:
It's undocumented. I had to go through next.js code to find it.